docs/releases/0.1.5.md

```diff
@@ -29,6 +29,11 @@ Refer to the [build documentation](../../build/build/#overdue-builds) for more i
 
 [#1202](https://github.com/inventree/InvenTree/pull/1202) introduces the concept of an *Expiry Date* for Stock Items. For further information, refer to the [expiry documentation](../../stock/expiry).
 
+### Stock Ownership
+
+[#1155](https://github.com/inventree/InvenTree/pull/1155) adds ownership to stock locations and items. Ownership allows to control which user group or individual user can edit and manage specific stock locations and items.
```

Comment:
`Ownership allows to control which user group .. `  would perhaps be better as `Ownership allows control over which user group ...`